lib/ansible/modules/cloud/openstack/os_nova_flavor.py

```diff
@@ -107,6 +112,18 @@
     cloud: mycloud
     state: absent
     name: tiny
+
+# Create flavor with metadata
```

Comment:
Please use `- name: Create flavor with metadata` (and remove `-` from in front of `os_nova_flavor:`